HacktoberfestProject.Web/Controllers/HomeController.cs

```diff
@@ -48,13 +49,27 @@ public async Task<IActionResult> Index()
         [HttpGet]
         public IActionResult Add()
         {
-            //How to access claims -left here for reference
-            ViewBag.Username = _contextAccessor.HttpContext.User
-                .Claims.FirstOrDefault(c => c.Type == GitHubUsernameClaimType).Value;
-
             return View();
         }
 
+        [Authorize]
+        [HttpPost]
+        public async Task<IActionResult> Add(AddPrViewModel vm)
+        {
+            if (!ModelState.IsValid)
+            {
+                return View("Add", vm);
+            }
+
+            vm.UserName = _contextAccessor.HttpContext.User

```

Comment:
what do you do it this is `null`?

Reply:
That is another question I have, what would we like an exception here/error page?

Reply:
If we can't get the username from the context then there is some other issue. @CrypticEngima do you think we would boot them to a page that would make them log in again?

Reply:
@CrypticEngima bump ^

Reply:
Agreed to jump to login page.